themes/digital.gov/layouts/shortcodes/img.html

```diff
@@ -58,25 +60,13 @@
         {{- end -}}"
       {{- end -}}
       {{- if eq $prefix $thisuid -}}
-        srcset="
-        {{- if ge $imgWidth 200 -}}
-          {{- $imgBaseCDN -}}_bu.jpg 48w,
-          {{- if ge $imgWidth 400 -}}
-            {{- if ge $imgWidth 800 -}}
-              {{- if ge $imgWidth 1200 -}}
-                {{- $imgBaseCDN -}}_w1200.{{- $imgExt }} 1200w,
-              {{- end -}}
-              {{- $imgBaseCDN -}}_w800.{{- $imgExt }} 800w,
-            {{- end -}}
-            {{- $imgBaseCDN -}}_w400.{{- $imgExt }} 400w,
-          {{- end -}}
-          {{- $imgBaseCDN -}}_w200.{{- $imgExt }} 200w
-        {{- else -}}
-          {{- $imgBaseCDN -}}.{{- $imgExt }} {{ $imgWidth -}}w
-        {{- end -}}
-        "
+        srcset=" {{- $imgBaseCDN -}}_bu.jpg 48w,
+        {{- $imgBaseCDN -}}_w200.{{- $imgExt }} 200w,
+        {{- $imgBaseCDN -}}_w400.{{- $imgExt }} 400w,
+        {{- $imgBaseCDN -}}_w800.{{- $imgExt }} 800w,
+        {{- $imgBaseCDN -}}_w1200.{{- $imgExt }} 1200w "
```

Comment:
determines image size based on viewport

Reply:
Looks good, I'm able to see the different sizes load from smaller to larger viewports.

<details><summary>Screenshots</summary>
<p>

<img width="1435" alt="Screenshot 2024-07-11 at 9 57 55 AM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/a0855894-d816-4040-be65-83a0b819398b">

<img width="1439" alt="Screenshot 2024-07-11 at 10 15 08 AM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/8aed3d27-1c27-4335-ab02-058bea91f825">


<img width="1417" alt="Screenshot 2024-07-11 at 10 14 36 AM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/cfdb0e1f-f3e4-4786-a3a4-6610b95d5119">

</p>
</details>